softlayer/vm/ubuntu.go

```diff
@@ -0,0 +1,324 @@
+package vm
```

Comment:
I called this `ubuntu.go` because I expected it would work with Ubuntu. I don't know that it should retain this name.